pinc/misc.inc

```diff
@@ -581,9 +581,12 @@ function endswith( $subject, $suffix )
     return ( substr( $subject, -strlen($suffix) ) == $suffix );
 }
 
-function str_contains( $haystack, $needle )
-{
-    return ( strpos( $haystack, $needle ) !== FALSE );
+if(!function_exists('str_contains')) {
+    // str_contains() now exists in PHP8
+    function str_contains( $haystack, $needle )
```

Comment:
this is available outside the if scope since if doesn't have scope in php, right?

Reply:
Yes, PHP functions are always global regardless of where they are defined unless they're in classes.